docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -7,7 +7,7 @@ lang: en
 
 Salesforce Code Builder is a second-generation (2GP) managed package that a user with requisite permissions can install in any supported Salesforce org edition.
 
-1. Go to  App Exchange and search for the Code Builder managed package.
+1.  Click the install [link](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N4V00000HddeKUAR) to access the Code Builder managed package on App Exchange.
```

Comment:
```suggestion
1.  Click the install [link](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N4V00000HddeKUAR) to access the Code Builder managed package on AppExchange.
```

Reply:
Not 100% sure if you should get rid of the space, but the main AppExchange page shows it that way so... https://appexchange.salesforce.com/